src/App.js

```diff
@@ -59,7 +61,11 @@ class Root extends React.Component {
     this.props.apiLoadErrorUpdate({apiLoadError: false});
 
     versionApi.getVersion().then((data) => {
-      hathorLibConfig.setNetwork(data.network.split('-')[0]);
+      let network = data.network;
+      if (data.network.includes('testnet')) {
+        network = 'testnet';
+      }
+      hathorLibConfig.setNetwork(network);
```

Comment:
The hathor-core is requiring the network to be `nano-testnet-alpha` for testing, so I had to support it with this change.

Reply:
I had to do this change also in the desktop wallet. I will create this method in the lib as @tuliomir suggested but I will change here later, otherwise I would need to wait for the lib PR before merging this one here.

When the new version is released I just update here the code.